### PR TITLE
DDCNL-2330 | Implement ATS tile on PTA

### PR DESCRIPTION
**Pull request by jpatton638**

### CODE REVIEW

**Comment by jpatton638** [comment]
Still waiting for the Welsh translation for the tile, due for COB 16/11

File: test/controllers/controllershelpers/HomeCardGeneratorSpec.scala [hunk 1]
```diff
@@ -52,10 +58,17 @@ class HomeCardGeneratorSpec extends ViewSpec with MockitoSugar {
...
       marriageAllowance,
-      statePension
+      statePension,
+      taxSummaries
     )
   val testUtr = SaUtr(new SaUtrGenerator().nextSaUtr.utr)
 
+  override implicit lazy val app: Application = new GuiceApplicationBuilder()
+    .configure(
+      "feature.tax-summaries-tile.enabled" -> true
```
Comment (struong) [inline test/controllers/controllershelpers/HomeCardGeneratorSpec.scala:68]: Don't need this block as it's true by default

File: app/controllers/controllershelpers/HomeCardGenerator.scala [hunk 2]
```diff
@@ -50,7 +51,8 @@ class HomeCardGenerator @Inject()(
       getTaxCalculationCard(taxCalculationStateCyMinusOne),
       getTaxCalculationCard(taxCalculationStateCyMinusTwo),
       getSelfAssessmentCard(saActionNeeded, currentTaxYear + 1),
-      getNationalInsuranceCard()
+      getNationalInsuranceCard(),
+      getAnnualTaxSummaryCard
```
Comment (struong) [inline app/controllers/controllershelpers/HomeCardGenerator.scala:55]: Has this got tests around it? (fine if not as its a bit of an odd function)

Comment (jpatton638) [reply app/controllers/controllershelpers/HomeCardGenerator.scala:55]: Each of the functions called here are tested, and this just returns the result of these functions, so it should be fine

**Review by struong — APPROVED** [review]

**Review by shikkhasingh — APPROVED** [review]

**Review by shikkhasingh — APPROVED** [review]

**Review by shikkhasingh — APPROVED** [review]

**Review by shikkhasingh — APPROVED** [review]

**Pull request merged**